modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -472,7 +472,10 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::checkInputWindow(const GenericReal<is_ad>
     if (behavior == WindowFailure::WARN)
       mooseWarning(msg.str());
     else if (behavior == WindowFailure::ERROR)
-      mooseError(msg.str());
+    {
+      msg << " \n   Cutting timestep.";
+      throw MooseException(msg.str());
```

Comment:
Also, this can just be `mooseException(mst.str())`. You may have to do some string handling here, but I think it should just work.

Reply:
I checked and the string is getting output.  Is that what you meant?

Reply:
Yeah that looks good.